.circleci/build.sh

```diff
@@ -156,9 +172,10 @@ echo "  > Installing Drupal into SQLite database ${DB_FILE}."
 "${BUILD_DIR}/vendor/bin/drush" -r "$(pwd)/${BUILD_DIR}/web" status
 
 echo "  > Symlinking module code."
-rm -rf "${BUILD_DIR}/web/modules/${MODULE}"/* > /dev/null
-mkdir -p "${BUILD_DIR}/web/modules/${MODULE}"
-ln -s "$(pwd)"/* "${BUILD_DIR}/web/modules/${MODULE}" && rm "${BUILD_DIR}/web/modules/${MODULE}/${BUILD_DIR}"
+rm -rf "${BUILD_DIR}/web/modules/custom"
+mkdir -p "${BUILD_DIR}/web/modules/custom"
```

Comment:
please remove this line - the line below uses `-p` and will create all required directories

Reply:
Updated it.